native/cocos/editor-support/spine/MathUtil.cpp

```diff
@@ -108,21 +65,23 @@ bool MathUtil::isNan(float v) {
     return _isNan(v, 0);
 }
 
-float MathUtil::random() {
-    return ::rand() / (float)RAND_MAX;
-}
-
-float MathUtil::randomTriangular(float min, float max) {
-    return randomTriangular(min, max, (min + max) * 0.5f);
-}
-
 float MathUtil::randomTriangular(float min, float max, float mode) {
     float u = random();
     float d = max - min;
     if (u <= (mode - min) / d) return min + sqrt(u * d * (mode - min));
     return max - sqrt((1 - u) * d * (max - mode));
 }
 
-float MathUtil::pow(float a, float b) {
-    return (float)::pow(a, b);
-}
+uint64_t MathUtil::ipow(uint64_t base, uint32_t exp) {
```

Comment:
Add an integer version of pow function. It's simpler and higher performance than the builtin float version.